### PR TITLE
First visuals

### PR DESCRIPTION
**Pull request by RNGezeus**
Identified all key elements and coloured. See ticket page for beta layout. 
TODO: Position things correctly. Currently dont handle resizing to small window well.

### CODE REVIEW

File: src/components/Groups.vue [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<template>
+    <div >
+        <h1> Grupper i F-spexet </h1>
+        <p class="bparagraph"> Lorem ipsum dolar lel </p> 
+            <h2>Styret</h2>
+            <p> 
+            Styret är de som ansvarar för att det överhuvudtaget blir ett F-spex. De sitter i otaliga lunchmöten för att planera, boka, styra, och se till att spexet fungerar och är roligt.
+            </p>
+
+                <h3> Orförande</h3>
```
Comment (feffes) [inline src/components/Groups.vue]: :thinking:

File: src/components/MainFooter.vue [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+
+            <h1>This is a footer,and now its gainning in lnength and losing typng strenth</h1> 
+
     </footer>
-</template>
\ No newline at end of file
+  </div>
+</template>
+
+<style>
```
Comment (feffes) [inline src/components/MainFooter.vue]: Might need scoped to only be applied to this element i.e
`<style scoped>`

File: src/components/MainHeader.vue [hunk 3]
```diff
@@ -1,20 +1,32 @@
...
+        <h1 class="link_text">Om oss</h1>
+      </router-link>
+      <router-link class="toplink" to="/contact">
+        <h1 class="link_text">Kontakt</h1>
+      </router-link>
+    </div>
+  </header>
 </template>
 
+<style>
```
Comment (feffes) [inline src/components/MainHeader.vue]: Same as comment on footer

Comment (feffes) [reply src/components/MainHeader.vue]: See https://vue-loader.vuejs.org/guide/scoped-css.html

File: src/components/MainFooter.vue [hunk 4]
```diff
@@ -1,5 +1,17 @@
 <template>
-    <footer>
-        <h1>This is a footer</h1>
+  <div id="bot">
```
Comment (feffes) [inline src/components/MainFooter.vue]: This seems to break https://github.com/fspexet/fspexet-vue-ts/blob/9be45e7964183732e2809c3bb693438ea194e413/src/app.scss#L9-L12
Is there a reason for this and if there is then that functionality should be applied to it as well.

**Comment by feffes** [comment]
force push after rebase onto master branch

File: src/components/Groups.vue (hunk 1 above)
Comment (RNGezeus) [reply src/components/Groups.vue]: Please explain, content copy pasted from old site.

File: src/components/MainFooter.vue (hunk 2 above)
Comment (feffes) [reply src/components/MainFooter.vue]: I suppose non-scoped styling could be used in conjunction with vue files to include certain stylings based on context. Probably not what we're looking for right now however.

File: src/components/Groups.vue (hunk 1 above)
Comment (feffes) [reply src/components/Groups.vue]: Or**d**förande

Comment (feffes) [reply src/components/Groups.vue]:
```suggestion
                <h3> Ordförande</h3>
```

Comment (RNGezeus) [reply src/components/Groups.vue]: The one thing I wrote before I started copy paste. I'm impressed with me. Language is hard.

Comment (feffes) [reply src/components/Groups.vue]: "impressed with _**myself**_." thank you very much.

File: src/components/MainFooter.vue (hunk 4 above)
Comment (RNGezeus) [reply src/components/MainFooter.vue]: Makes no sense and replaced with things that actually has meaning.

**Comment by feffes** [comment]
There are some changes to usage of `<p>` tags in tickets component that are addressed in #3 , it should be merged into this branch before merge into master.

**Pull request merged**